doc/articles/get-started-vs-2022.md

```diff
@@ -93,26 +90,35 @@ To create an Uno Platform app:
     - Press the `MyApp.Server` button to deploy the app
 1. To debug for **iOS**:
     - Right click on the `MyApp.Mobile` project, select **Set as startup project**
+        > [!NOTE]
+        > For information about connecting Visual Studio to a Mac build host to build iOS apps, see [Pairing to a Mac for .NET iOS development](https://docs.microsoft.com/en-us/xamarin/ios/get-started/installation/windows/connecting-to-mac/).
+
     - In the "Debug toolbar" drop-down, select framework `net7.0-ios`:
 
       ![Visual Studio - "Debug toolbar" drop-down selecting the "net7.0-ios" framework](Assets/quick-start/net7-ios-debug.png)
       
     - Select:
       - An active device, if your IDE is connected to a macOS Host
       - A [local device using Hot Restart](https://learn.microsoft.com/en-us/xamarin/xamarin-forms/deploy-test/hot-restart), to debug your application without connecting to a mac
+      
+        > [!NOTE] 
+        > If no iOS devices are available, a Visual Studio 17.7+ issue requires unloading/reloading the project. Right-click on the `.Mobile` project and select **Unload Project** then **Load project**.
+
 1. To debug the **Android** platform:
     - Right click on the `MyApp.Mobile` project, select **Set as startup project**
     - In the **Debug toolbar** drop down, select framework `net7.0-android`
     - Select an active device in the "Device" sub-menu
+        > [!NOTE] 
+        > If no android devices are available, a Visual Studio 17.7+ issue requires unloading/reloading the project. Right-click on the `.Mobile` project and select **Unload Project** then **Load project**.
```

Comment:
@jeromelaban  do we need to share the link to the related Visual Studio 17.7+ issue ?